src/components/YahooSellNotification.js

```diff
@@ -0,0 +1,124 @@
+import React, { useState } from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+import Button from '@material-ui/core/Button'
+import SetUserSearchEngineMutation from 'src/utils/mutations/SetUserSearchEngineMutation'
+import SetYahooSearchOptInMutation from 'src/utils/mutations/SetYahooSearchOptInMutation'
+import CreateSearchEnginePromptLogMutation from 'src/utils/mutations/CreateSearchEnginePromptLogMutation'
+import PropTypes from 'prop-types'
+import { Typography } from '@material-ui/core'
+import Notification from './Notification'
+
+const useStyles = makeStyles((theme) => ({
+  learnButton: {
+    height: '30px',
+    fontWeight: '900',
+    fontFamily: 'Poppins',
+  },
+  noThanksButton: {
+    marginLeft: 'auto',
+    borderRadius: '15px',
+    height: '30px',
+    fontWeight: '900',
+    fontFamily: 'Poppins',
+  },
+  switchToYahooButton: {
+    background: '#29BEBA',
+    borderRadius: '15px',
+    height: '30px',
+    fontWeight: '900',
+    fontFamily: 'Poppins',
+    marginLeft: theme.spacing(1),
+  },
```

Comment:
The "switch to yahoo" button text should be white:
https://6102cd7112cc720039c0b2da-zlxphgosuc.chromatic.com/?path=/story/components-yahoosellnotification--basic&globals=backgrounds.grid:false